tree/ntuple/v7/src/RField.cxx

```diff
@@ -1671,9 +1680,63 @@ void ROOT::Experimental::RVariantField::CommitCluster()
    std::fill(fNWritten.begin(), fNWritten.end(), 0);
 }
 
-
 //------------------------------------------------------------------------------
 
+std::string ROOT::Experimental::RPairField::RPairField::GetTypeList(
+   const std::vector<std::unique_ptr<Detail::RFieldBase>> &itemFields)
+{
+   return itemFields[0]->GetType() + "," + itemFields[1]->GetType();
```

Comment:
NB: Perhaps we should rename `RField::GetName()` and `RField::GetType()` to `GetFieldName()` and `GetTypeName()`? Not for this PR though.